handlers.py

```diff
@@ -72,7 +111,24 @@ def handle_connect_wifi(server, body, addr):
         print("error running connect-wifi.sh: {0}".format(str(err)))
         raise Exception("Failed to connect to {0} network".format(ssid))
 
-    # Respond to client with an ACK to confirm the connection was successful
-    server.send(json.dumps({'type': MsgType.ACK}).encode('utf-8'), addr)
+    # Respond to client with an ACK to confirm the connection was successful.
+    # Retry if client sends the WIFI_CONN message within 10 seconds to indicate
+    # that the ACK message was dropped.
+    server.socket.settimeout(10)
+    while True:
```

Comment:
I think we should limit this to 3 attempts, so that it doesn't end up waiting forever.